cost-analyzer/templates/forecasting-deployment.yaml

```diff
@@ -0,0 +1,124 @@
+{{- if .Values.forecasting.enabled }}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "forecasting.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{ include "forecasting.commonLabels" . | nindent 4 }}
+    {{- with .Values.global.additionalLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{ include "forecasting.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: forecasting
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app: forecasting
+      {{- with .Values.global.podAnnotations}}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+
+    spec:
+      {{- if .Values.global.platforms.openshift.enabled }}
+      securityContext:
+      {{- toYaml .Values.global.platforms.openshift.securityContext | nindent 8 }}
+      {{- else if .Values.global.securityContext }}
+      securityContext:
+      {{- toYaml .Values.global.securityContext | nindent 8 }}
+      {{- else }}
+      securityContext:
+        runAsUser: 1001
+        runAsGroup: 1001
+        fsGroup: 1001
+      {{- end }}
+
+      restartPolicy: Always
+
+      volumes:
+      - name: models
+        {{/*
+        An emptyDir for models is necessary because of the
+        readOnlyRootFilesystem default In the future, this may optionally be a
+        PV
+        */}}
+        emptyDir: {}
+
+      containers:
+        - name: forecasting
+          {{- if .Values.forecasting.fullImageName }}
+          image: {{ .Values.forecasting.fullImageName }}
+          {{- else }}
+          image: gcr.io/kubecost1/forecasting:prod-{{ $.Chart.AppVersion }}
+          {{ end }}
+          args: ["forecasting"]
+          {{/*
+          fix readinessProbe
+
+          readinessProbe:
+            httpGet:
+              path: /healthz
+              port: 5000
+            initialDelaySeconds: 10
+            periodSeconds: 5
+            failureThreshold: 200
+          */}}
```

Comment:
Commented out. Need to fix.

Reply:
@CampfireRemnants can you make flask serve `/healthz` and always return a 200? That's a prerequisite

Reply:
Yes, easily.

Reply:
Probes fixed. Will test once an image with /healthz is available.